prow/cluster/configmaps/config.yaml

```diff
@@ -150,6 +150,7 @@ presubmits:
 
   istio/test-infra:
   - name: test-infra-presubmit
+    skip_report: true
```

Comment:
why ? I'm curious to see how prow reports work and I hope it doesn't have the bug that our jenkins report has of posting a comment that triggers a lot of useless github notifications ?